bitnami/keycloak/templates/configmap-env-vars.yaml

```
@@ -35,7 +35,7 @@ data:
       {{- include "common.tplvalues.render" (dict "value" $path "context" $) }}
     {{- end }}
   {{- end }}
-  {{- if and .Values.ingress.enabled .Values.ingress.hostname }}
+  {{- if and .Values.ingress.enabled (and .Values.ingress.hostname (not (empty .Values.proxyHeaders))) }}
```

From the description I understand this line should be:

```suggestion
  {{- if and .Values.ingress.enabled .Values.ingress.hostname (empty .Values.proxyHeaders) }}
```

If `proxyHeaders` is set, `KEYCLOAK_HOSTNAME` shouldn't be filled.

---

I have updated the code.